### PR TITLE
Implement gRPC Team Add-User

### PR DESCRIPTION
**Pull request by drgarcia1986**
* Implement team add-user
* Remove go-swagger implementation
* Clean up
* love ❤️

---
This change is [Reviewable](https://reviewable.io/reviews/luizalabs/teresa-api/193)

### CODE REVIEW

File: cmd/client/cmd/team.go [hunk 1]
```diff
@@ -140,3 +124,27 @@ func createTeam(cmd *cobra.Command, args []string) {
 	}
 	fmt.Println("Team created with success")
 }
+
+func teamAddUser(cmd *cobra.Command, args []string) {
+	team, _ := cmd.Flags().GetString("team")
```
Comment (arnaldopereira) [inline cmd/client/cmd/team.go]: Shouldn't we test for errors here?

Comment (drgarcia1986) [inline cmd/client/cmd/team.go]: copy/past

File: pkg/server/team/handlers_test.go [hunk 2]
```diff
@@ -55,3 +56,89 @@ func TestTeamCreateTeamAlreadyExists(t *testing.T) {
...
+	if len(teamWithUser.Users) == 0 {
+		t.Fatal("AddUser dont add user for a team")
+	}
+
+	for _, u := range teamWithUser.Users {
+		if u.Email == expectedEmail {
+			return
+		}
+	}
+	t.Errorf("AddUser dont add user for a team")
```
Comment (aguerra) [inline pkg/server/team/handlers_test.go]: Maybe `AddUser didn't add user in team`

File: cmd/client/cmd/team.go [hunk 3]
```diff
@@ -140,3 +124,27 @@ func createTeam(cmd *cobra.Command, args []string) {
...
+	user, _ := cmd.Flags().GetString("user")
+	if team == "" || user == "" {
+		cmd.Usage()
+		return
+	}
+
+	conn, err := connection.New(cfgFile)
+	if err != nil {
+		fmt.Fprintln(os.Stderr, "Erro connecting to server:", err)
+		return
```
Comment (aguerra) [inline cmd/client/cmd/team.go]: Typo Erro -> Error

**Pull request merged**